src/plugins/intel_cpu/src/nodes/executors/mvn_list.hpp

```diff
@@ -7,6 +7,9 @@
 #include "executor.hpp"
 
 #include "mvn.hpp"
+#if defined(OV_CPU_WITH_ACL)
+#include "acl/acl_mvn.hpp"
+#endif
 #include "x64/jit_mvn.hpp"
 #include "common/ref_mvn.hpp"
```

Comment:
looks like there is no need to include executors in global header

Reply:
Executors are used in `mvn_list,cpp` to declare the list. So, executor headers need to be included in either `mvn_list.cpp` or `mvn_list.hpp`
I followed Dmitry's code, matmul and eltwise executor headers are included in hpp.

Reply:
Lets leave as is. I will refactor this in bounds of static factory introduction